src/app/clients/clients-view/view-charge/view-charge.component.html

```diff
@@ -0,0 +1,149 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary" [routerLink]="['../../','paycharge',chargeData.id]">
+    <fa-icon icon="dollar-sign"></fa-icon>&nbsp;&nbsp;
+    Pay
+  </button>
+  <button mat-raised-button color="primary" (click)="waiveCharge()">
+    <i class="fa fa-flag"></i>
+    Waive Charge
+  </button>
+  <button mat-raised-button color="primary" (click)="deleteCharge()">
+    <fa-icon icon="trash"></fa-icon>&nbsp;&nbsp;
+    Delete
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+    
+    <mat-card-title> <i class="fa fa-stop" [ngClass]="(!(chargeData.isWaived || chargeData.isPaid))|statusLookup"></i> 
+      {{ chargeData.name }}
+    </mat-card-title>
+
+    <mat-divider></mat-divider>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" class="content" style="padding-top:15px">
+
```

Comment:
![Screenshot from 2020-05-13 12-19-22](https://user-images.githubusercontent.com/54709463/81780370-0ab8f700-9514-11ea-9275-bfbeee49f392.png)
make rows look like this